boards/arm/nucleo_wb55rg/Kconfig.defconfig

```diff
@@ -10,8 +10,14 @@ if BOARD_NUCLEO_WB55RG
 config BOARD
 	default "nucleo_wb55rg"
 
-config CLOCK_STM32_LSE
```

Comment:
Why removing this ?

Reply:
restore config CLOCK_STM32_LSE in boards/arm/nucleo_wb55rg/Kconfig.defconfig